addons/website/views/snippets/snippets.xml

```diff
@@ -938,7 +941,7 @@
     <t t-set="only_bg_image_exclude" t-value="''"/>
```

Comment:
Generic question: we have an "invert color" for indicators... but it is not controllable depending on the slide background color. Should not we make something generic that automatically invert the color depending on the active slide? cc @lebl-odoo

Reply:
It could be a nice to have, I'll add it to my notes. Thx